quiz/backend/players/models.py

```diff
@@ -0,0 +1,40 @@
+import uuid
+from django.db import models
+
+
+class Player(models.Model):
```

Comment:
Model is not compatible with ER diagram. Should we update the diagram or amend the model?

Reply:
I think it is better to update ER Diagram, because provided in story was obsolete.
https://azeno.youtrack.cloud/articles/AQ-A-1/Diagram-bazy-danych-do-aplikacji-Azeno-Quiz